docs/core/deploying/trim-warnings/il2060.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "IL2060: Call to 'System.Reflection.MethodInfo.MakeGenericMethod' cannot be statically analyzed by the trimmer"
+description: "Learn about trim warning IL2060: MakeGenericMethodCall"
+ms.date: 07/23/2021
+ms.topic: reference
+author: mateoatr
+ms.author: matorre
+f1_keywords:
+  - "IL2060"
+---
+# IL2060: Call to 'System.Reflection.MethodInfo.MakeGenericMethod' cannot be statically analyzed by the trimmer
+
+## Cause
+
+Calls to <System.Reflection.MethodInfo.MakeGenericMethod> cannot be statically analyzed by the trimmer.
```

Comment:
```suggestion
Call to <System.Reflection.MethodInfo.MakeGenericMethod> cannot be statically analyzed by the trimmer.
```

trimmer can figure out some of them.. so the specific call on which this was reported is problematic.